core-mapping/build.gradle

```diff
@@ -1,27 +1,25 @@
-apply plugin : 'java-library'
+plugins {
+	id 'com.github.johnrengelman.shadow' version '8.1.1'
```

Comment:
Do we really need it? It is usually used to produce a fat jar

Reply:
I don't think we do. I noticed there was a custom config to achieve the same, but I think we can simply remove it

Reply:
[6649061](https://github.com/MrStahlfelge/gdx-controllerutils/pull/29/commits/664906111caaae23d415b7730db2469468ab5c43)